cypress-custom/support/commands.js

```diff
@@ -3,3 +3,15 @@ Cypress.Commands.add('swapSelectInput', tokenAddress => {
   cy.get('.token-item-' + tokenAddress).should('be.visible')
   cy.get('.token-item-' + tokenAddress).click({ force: true })
 })
+
+function _responseHandlerFactory(body) {
+  return req =>
+    req.reply(res => {
+      const newBody = JSON.stringify(body || res.body)
+      res.body = newBody
+    })
+}
+
+Cypress.Commands.add('swapStubResponse', ({ query, alias = 'stubbedResponse', body }) => {
```

Comment:
nice. Maybe u can loose the "swap" word, since it's general porpouse. Is not doing anything in the swap ui, or swap page

Reply:
true!